docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Please add a link to the DualPotentials class as {class}<code>dual potentials &lt;ott.problems.linear.potentials.DualPotentials&gt;</code></p><p>Also please add a link to the transport method (using {meth}).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>